apps/docs/src/app/core/component-docs/dialog/examples/dialog-inner-popover/dialog-inner-popover.component.html

```diff
@@ -0,0 +1,47 @@
+<fd-dialog>
```

Comment:
Do we need this file? In the appropriate TS file i don't see `templateUrl` to this file, but only inline `template`.